scene/gui/scroll_container.cpp

```diff
@@ -139,7 +139,7 @@ void ScrollContainer::_gui_input(const Ref<InputEvent> &p_gui_input) {
 				drag_accum = Vector2();
 				last_drag_accum = Vector2();
 				drag_from = Vector2(h_scroll->get_value(), v_scroll->get_value());
-				drag_touching = OS::get_singleton()->has_touchscreen_ui_hint();
+				drag_touching = true;
```

Comment:
The `if (drag_touching)` condition below can be removed, given that it is set to true in the same scope.

Reply:
Setting this to `true` unconditionally sounds like it might break compatibility though, I'm not sure if that's the desired behavior.